rfhub2/cli/population.py

```diff
@@ -0,0 +1,175 @@
+# from pathlib import Path
+import robot.libraries
+from robot.errors import DataError
+from robot.libdocpkg import LibraryDocumentation
+from requests import session, post
+from rfhub2.config import APP_INTERFACE, APP_PORT, BASIC_AUTH_USER, BASIC_AUTH_PASSWORD
```

Comment:
Let's stick to convention where you have first group of external imports in alphabetical order and then, after one empty line internal, rfhub imports